OutOfSchool/OutOfSchool.WebApi/OutOfSchool.WebApi.csproj

```diff
@@ -12,12 +12,15 @@
         <CodeAnalysisRuleSet>..\StyleCopAnalyzersRules.ruleset</CodeAnalysisRuleSet>
         <ImplicitUsings>enable</ImplicitUsings>
     </PropertyGroup>
+    <ItemGroup>
+      <Compile Remove="Models\Workshop\WorkshopUpdateDTO.cs" />
+    </ItemGroup>
```

Comment:
Please check if this line is necessary. 
Usually it is better to remove file from folder, than to have a dead code that is never touched by compiler because of this `Compile Remove` directive

Reply:
Fixed.